IMOS/imosParameters.txt

```diff
@@ -53,11 +53,13 @@ DIR_MAG,             0, from_direction,
 DIRECTION,           0, direction_of_the_profile,                                                                 ,              ,              ,                                  ,  ,         ,         ,         char
 DIRT,                0, to_direction,                                                                             degree,        clockwise,     true north,                        E, 999999.0, 0.0,      360.0,    float
 DIST_ALONG_BEAMS,    0, distance_from_sensor_along_beams,                                                         m,             ,              sensor,                            Z, 999999.0, -12000.0, 12000.0,  float
-DOX1,                1, mole_concentration_of_dissolved_molecular_oxygen_in_sea_water,                            umol l-1,      ,              ,                                  O, 999999.0, 0.0,      900000.0, float
-DOX2,                1, moles_of_oxygen_per_unit_mass_in_sea_water,                                               umol kg-1,     ,              ,                                  O, 999999.0, 0.0,      880000.0, float
+DOX,                 0, volume_concentration_of_dissolved_molecular_oxygen_in_sea_water,                          ml l-1,        ,              ,                                  O, 999999.0, 0.0,      200.0,    float
+DOX1,                1, mole_concentration_of_dissolved_molecular_oxygen_in_sea_water,                            umol l-1,      ,              ,                                  O, 999999.0, 0.0,      1000.0, float
+DOX2,                1, moles_of_oxygen_per_unit_mass_in_sea_water,                                               umol kg-1,     ,              ,                                  O, 999999.0, 0.0,      1000.0, float
 DOXS,                1, fractional_saturation_of_oxygen_in_sea_water,                                             percent,       ,              ,                                  O, 999999.0, ,         ,         float
 DOXY,                1, mass_concentration_of_oxygen_in_sea_water,                                                kg m-3,        ,              ,                                  O, 999999.0, 0.0,      29.0,     float
 DOXY_TEMP,           1, temperature_of_sensor_for_oxygen_in_sea_water,                                            degrees_Celsius,,             ,                                  T, 999999.0, 0.0,      50.0,     float
+OXSOL,               1, moles_of_oxygen_per_unit_mass_in_sea_water_at_saturation,                                 umol kg-1,     ,              ,                                  O, 999999.0, 0.0,      1000.0, float

```

Comment:
"Is CF parameter" code should be 0, not 1. 
I could not find a CF standard name moles_of_oxygen_per_unit_mass_in_sea_water_at_saturation in http://cfconventions.org/Data/cf-standard-names/45/build/cf-standard-name-table.html . However, could this be a synonym of apparent_oxygen_utilization ("Apparent Oxygen Utilization (AOU) is the difference between measured dissolved oxygen concentration in water, and the equilibrium saturation concentration of dissolved oxygen in water with the same physical and chemical properties") ?